wfe/web-front-end.go

```diff
@@ -326,39 +316,44 @@ const (
 )
 
 // verifyPOST reads and parses the request body, looks up the Registration
-// corresponding to its JWK, verifies the JWS signature,
-// checks that the resource field is present and correct in the JWS protected
-// header, and returns the JWS payload bytes, the key used to verify, and the
-// corresponding Registration (or error).
-// If regCheck is false, verifyPOST will still try to look up a registration
-// object, and will return it if found. However, if no registration object is
-// found, verifyPOST will attempt to verify the JWS using the key in the JWS
-// headers, and return the key plus a dummy registration if successful. If a
-// caller passes regCheck = false, it should plan on validating the key itself.
-func (wfe *WebFrontEndImpl) verifyPOST(request *http.Request, regCheck bool, resource core.AcmeResource) ([]byte, *jose.JsonWebKey, core.Registration, error) {
+// corresponding to its JWK, verifies the JWS signature, checks that the
+// resource field is present and correct in the JWS protected header, and
+// returns the JWS payload bytes, the key used to verify, and the corresponding
+// Registration (or error).  If regCheck is false, verifyPOST will still try to
+// look up a registration object, and will return it if found. However, if no
+// registration object is found, verifyPOST will attempt to verify the JWS using
+// the key in the JWS headers, and return the key plus a dummy registration if
+// successful. If a caller passes regCheck = false, it should plan on validating
+// the key itself.  verifyPOST also appends its errors to requestEvent.Errors so
+// code calling it does not need to if they imediately return a response to the
+// user.
+func (wfe *WebFrontEndImpl) verifyPOST(e *requestEvent, request *http.Request, regCheck bool, resource core.AcmeResource) ([]byte, *jose.JsonWebKey, core.Registration, error) {
```

Comment:
Here and in the below, I would prefer a more expressive name than `e`, say `event` or `reqEvent`.